c_src/main.c

```diff
@@ -119,7 +111,6 @@ erlfdb_create_future(ErlNifEnv* env, FDBFuture* future, ErlFDBFutureGetter gette
     f->future = future;
     f->fgetter = getter;
     enif_self(env, &(f->pid));
-    f->pid_env = env;
```

Comment:
https://erlang.org/documentation/doc-5.10.2/erts-5.10.2/doc/html/erl_nif.html

> It is thus useless and dangerous to store pointers to process bound environments between NIF calls.

If the process that created the nif is over, the env may be freed, but we will still have a pointer to it and try to use in `enif_send`; that causes segfaults occasionally.